azkaban-common/src/main/java/azkaban/executor/container/KubernetesContainerizedImpl.java

```diff
@@ -83,6 +83,10 @@
       "/export/apps/azkaban/azkaban-exec-server/current/plugins/jobtypes";
   public static final String IMAGE = "image";
   public static final String VERSION = "version";
+  public static final String NSCD_SOCKET_VOLUME_NAME = "nscd-socket";
+  public static final String NSCD_SOCKET_HOST_PATH = "/var/run/nscd/socket";
```

Comment:
Should we have this path passed from the Azkaban properties? 
If this is the standard path for nscd even outside LinkedIn, then we can keep it.

Reply:
You are right. Fixed it. Thanks.